_pages/about.html

```diff
@@ -8,11 +8,95 @@
     <div class="page-header">
         <h1>About Us</h1>
     </div>
-    {% for group in site.members %}
-	    <div class="container">
-	    	<table style="width:100%" id="about-table">
-				{{ group }}
-			</table>
-    	</div>
-    {% endfor %}
-</body>
+    <div class="container">
+    	<table style="width:100%" id="about-table">
+            {% assign num_members = -1 %}
+            {% assign per_row = 3 %}
+            {% for member in site.data.members %}
+                {% assign num_members = num_members | plus: 1 %}
+                {% assign mod = num_members | modulo:per_row %}
+                {% if mod == 0 %}
+                 <div class="row">
+                {% endif %}
+                <div class="col col-lg-4 col-md-4 col-sm-12">
+                    <center>
+                    <div class="hovereffect">
+                        <center>
+                            <div class="crop">
+                            <img class="img-responsive" src="/static/images/members/{{ member.name }}" alt="" width="300" height="300">
```

Comment:
Use a background div image so you can have it zoom-fill: https://stackoverflow.com/a/11763434